linkerd/proxy/http/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+
+[package]
+name = "linkerd2-proxy-http"
+version = "0.1.0"
+authors = ["Linkerd Developers <cncf-linkerd-dev@lists.cncf.io>"]
+edition = "2018"
+publish = false
+description = """
+HTTP-specific implementations that rely on other proxy infrastructure
+
+This should probably be decomposed into smaller, decoupled crates.
```

Comment:
Is there a reason this is necessary? Does this crate take a long time to build?

Reply:
It's just a big bucket of unrelated utilities and shouldn't have any other dependencies on linkerd2-proxy things, except it needs transport::Source at the moment.